src/js/themes/base.js

```diff
@@ -614,6 +614,9 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         // extend: undefined,
       },
     },
+    checkBoxGroup: {
+      // container: {}, // any box props
```

Comment:
For consistency, are we using the `// any box props` comment anywhere else? Haven't noticed it as a convention, but do think it is helpful. Should `extend` be documented as well?

```suggestion
      // container: {
        // any box props
        // extend: undefined,
      //},
```

Reply:
We used it in the radioButtonGroup.container so I had carried it over from there.

Reply:
I think it would be beneficial to document extend.